handlers/grafana_handlers.go

```diff
@@ -97,6 +118,13 @@ func (h *Handler) GrafanaPingHandler(w http.ResponseWriter, req *http.Request, p
 	_, _ = w.Write([]byte("{}"))
 }
 
+// swagger:route GET /api/telemetry/metrics/grafana/boards GrafanaAPI idGetGrafanaBoards
+// Handle GET request for Grafana boards
+//
+// Used for fetching Grafana boards and panels
+// responses:
+// 	200: grafanaBoardsResponseWrapper
+
```

Comment:
This endpoint also supports a query `dashboardSearch`, to search for specific boards. Do we include that in the swagger schema?